lib/posix/posix.nim

```diff
@@ -448,7 +448,7 @@ type
     sa_family*: TSa_Family         ## Address family.
     sa_data*: array [0..255, char] ## Socket address (variable-length data).
 
-  Tsockaddr_storage* {.importc: "struct sockaddr_storage",
+  Sockaddr_storage* {.importc: "struct sockaddr_storage",
```

Comment:
I thought I did it in line 584: https://github.com/nim-lang/Nim/pull/2976/files#diff-880812f441788508a84adff2c2ead03dR584

Did I do it wrongly?